internal/cli/fs.go

```diff
@@ -0,0 +1,56 @@
+package cli
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+// Mkdir makes the directories specified by path
+func Mkdir(path string) error {
+	if err := os.MkdirAll(path, os.ModePerm); err != nil {
+		return fmt.Errorf("failed to create directory at %s: %w", path, err)
+	}
+	return nil
+}
+
+// WriteFile writes data to the specified filepath
+func WriteFile(path string, perm os.FileMode, r io.Reader) error {
+	f, openErr := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, perm)
+	if openErr != nil {
+		return fmt.Errorf("failed to open file at %s: %s", path, openErr)
+	}
+	defer f.Close()
+
+	if _, err := io.Copy(f, r); err != nil {
+		return fmt.Errorf("failed to write file at %s: %w", path, err)
+	}
+	return nil
+}
+
+// WriteZip writes the zip contents to the specified filepath
+func WriteZip(basePath string, zipPkg *zip.Reader) error {
+	for _, zipFile := range zipPkg.File {
+		fmt.Println("opening zip file:", basePath, zipFile.Name)
+		path := filepath.Join(basePath, zipFile.Name)
+
+		if zipFile.FileInfo().IsDir() {
+			if err := Mkdir(path); err != nil {
+				return err
+			}
+		} else {
```

Comment:
[nit] could just `return Mkdir(path)` here or at the very least return early here and drop the else

Reply:
not sure I follow, this is done inside a for loop of all the zip's files.  I _think_ we still want to be iterating through the rest of the package, right?

Reply:
yes you're right. I meant to do a continue here instead of a return.

```if zipFile.FileInfo().IsDir() {
			if err := Mkdir(path); err != nil {
				return err
			}
		}
continue
```
that way we can drop the else and the nested block

Reply:
roger that